common/app/utils/form.js

```diff
@@ -65,6 +65,10 @@ export function getValidationState(field) {
     return null;
   }
 
+  if ((/http(s):\/\/glitch\.com\/edit\/#!\/.*/g).exec(field.value)) {
+    return 'warning';
```

Comment:
Can we make this less generic. It should return something like 'glitch-warning', so we can differentiate it from other warnings we might add in the future.

Reply:
The reason why I used `warning` is because of React Bootstrap. The only three validation states in a form are success, error, and warning. 

I could catch a `glitch-warning` and turn it into a `warning` state within the Solution Input component. Although this is almost the same flow but reordered, it would increase the level of specificity within `getValidationState`. Would you like me to implement that?